cheat/cheatsheets/git

```diff
@@ -17,9 +17,16 @@ git commit -m "Your commit message"
 # To edit previous commit message
 git commit --amend
 
+# Git commit in the past
+git commit --date="`date --date='2 day ago'`"
```

Comment:
Doesn’t work in Debian Wheezy’s Git 1.7.10.4, but that’s not really relevant since Wheezy’s outdated. Does anyone have a Jessie available?